src/components/dropdown/UIDropdown.vue

```diff
@@ -7,33 +7,35 @@
               :show="show"
               :hide="hide" />
 
-        <template v-if="transitionComponent">
-            <component :is="transitionComponent">
-                <div v-if="isOpen"
-                     v-bind="$attrs"
-                     ref="dropdown"
-                     v-click-away="hide"
-                     class="dropdown absolute z-50 rounded overflow-auto flex flex-col items-stretch shadow-lg
+        <Teleport to="body">
+            <template v-if="transitionComponent">
```

Comment:
Should be able to move this conditional to the `component` line below, tidy up the HTML a little.

Reply:
If I remember correctly I could not and there was a reason why i hed the `<template>` tag.
I think the `<component>` needs to have it's dependency (TransitionComponent) known when mounting if that makes sense.

Reply:
Doesn't seem to be needed anymore or I'm not testing some edge case. Removing.